modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/events/SegmentsServicePreAction.java

```diff
@@ -170,7 +170,7 @@ protected void doRun(
 							groupId, User.class.getName(), userId, context);
 
 					_segmentsEntrySessionCache.putSegmentsEntryIds(
-						segmentsEntryIds);
+						groupId, segmentsEntryIds);
 				}
 				else {
```

Comment:
Do we need this if/else method?

Reply:
not at all, refactoring in a separate commit as @jorgediaz-lr requested